### ISSUE TITLE
Cannot run file scan without pcntl module

### ISSUE THREAD

**Issue by PVince81** [owncloud/core#22001]
Run occ files:scan on openSUSE Tumbleweed without having the pcntl module:
```
PHP Fatal error:  Call to undefined function OCA\Files\Command\pcntl_signal() in /srv/www/htdocs/owncloud/apps/files/command/scan.php on line 206
PHP Stack trace:
PHP   1. {main}() /srv/www/htdocs/owncloud/occ:0
PHP   2. require_once() /srv/www/htdocs/owncloud/occ:11
PHP   3. OC\Console\Application->run() /srv/www/htdocs/owncloud/console.php:81
PHP   4. Symfony\Component\Console\Application->run() /srv/www/htdocs/owncloud/lib/private/console/application.php:110
PHP   5. Symfony\Component\Console\Application->doRun() /srv/www/htdocs/owncloud/3rdparty/symfony/console/Application.php:123
PHP   6. Symfony\Component\Console\Application->doRunCommand() /srv/www/htdocs/owncloud/3rdparty/symfony/console/Application.php:192
PHP   7. Symfony\Component\Console\Command\Command->run() /srv/www/htdocs/owncloud/3rdparty/symfony/console/Application.php:840
PHP   8. OCA\Files\Command\Scan->execute() /srv/www/htdocs/owncloud/3rdparty/symfony/console/Command/Command.php:259
PHP   9. OCA\Files\Command\Scan->initTools() /srv/www/htdocs/owncloud/apps/files/command/scan.php:162
```

@DeepDiver1975 @mmattel did we merge the wrong version of the PR ?

**Comment by PVince81** [comment]
On master 1765e11f7573e61ae4506111ca60a51818a3788a

**Comment by PVince81** [comment]
Okay, seems this was broken already on master since the statistics PR.

This PR fixes it: https://github.com/owncloud/core/pull/21701

**Comment by mmattel** [comment]
#21701 is not merged - it is ready to merge...

**Comment by PVince81** [comment]
Yeah sorry, I thought we merged a wrong version of it

**Comment by rullzer** [comment]
Should be fixed now that #21701 is in.
